lib/strong_migrations/migration_helpers.rb

```diff
@@ -73,16 +75,172 @@ def remove_null_constraint_safely(table_name, column_name, name: nil)
       end
     end
 
+    def add_column_safely(table_name, column_name, type, **options)
+      ensure_postgresql(__method__)
+      ensure_not_in_transaction(__method__)
+
+      default = options.delete(:default)
+
+      if postgresql_version >= 110000 || default.nil?
+        add_column(table_name, column_name, options)
+      else
+        reversible do |dir|
+          dir.up do
+            transaction do
+              add_column(table_name, column_name, type, default: nil, **options)
+              change_column_default(table_name, column_name, default)
+            end
+
+            default_after_type_cast = connection.type_cast(default)
```

Comment:
How hard would it be to support default expression e.g. `uuid_generate_v4()`? We've needed to do that pretty often as we've migrated away from `bigint` identifiers (at least for what we externally expose).

Reply:
It is not *that* hard, but it definitely will introduce some ugly code changes in `backfill_column_safely` method. Let's wait for more demand and implement it in separate PR, if needed.